src/signature.js

```diff
@@ -270,8 +270,11 @@ const shortName = R.compose(
   ensureParametrized,
 );
 
-// :: [Type] -> TypeMap
-const indexTypes = R.indexBy(shortName);
+// :: [(Type -> Type)] -> [Type] -> TypeMap
```

Comment:
Shouldn’t it be `:: [(*... -> Type)] -> [Type] -> TypeMap` ?

Reply:
More likely, `(Type... -> Type)`. But I'm not sure that this is a normal signature of a variable number of arguments of a certain type. What do you think?

Reply:
What is normal? :) I remember Ramda uses ellipsis extensively